0051-v1beta1-schema/README.md

```diff
@@ -0,0 +1,546 @@
+        namespace: podinfo-from-oci
+        oci:
+          url: oci://ghcr.io/stefanprodan/charts/podinfo
+          version: 6.4.0
+        valuesFiles:
+          - values.yaml
+
+      - name: podinfo-git
+        namespace: podinfo-from-git
+        git:
+          url: https://github.com/stefanprodan/podinfo.git@6.4.0
+          path: charts/podinfo  # Changed from `gitPath`
+        # version field removed - uses version from chart.yaml at git tag
+        valuesFiles:
+          - values.yaml
+
+      - name: podinfo-repo
+        namespace: podinfo-from-repo
+        helmRepo:
+          url: https://stefanprodan.github.io/podinfo
+          name: podinfo  # Changed from `repoName`
+          version: 6.4.0
+        releaseName: cool-release-name
+        valuesFiles:
+          - values.yaml
+```
+
+#### Story 2
+
+As a user of health checks in my package, I have the following `zarf.yaml` in v1alpha1:
+
+```yaml
+kind: ZarfPackageConfig
+metadata:
+  name: health-checks
+  description: Deploys a simple pod to test health checks
+
+components:
+  - name: health-checks
+    required: true
+    manifests:
+      - name: ready-pod
+        namespace: health-checks
+        noWait: true
+        files:
+          - ready-pod.yaml
+    healthChecks:
+      - name: ready-pod
+        namespace: health-checks
+        apiVersion: v1
+        kind: Pod
+```
+
+I want to move to the latest schema, so I run `zarf dev convert`, which produces:
+
+```yaml
+apiVersion: v1beta1
+kind: ZarfPackageConfig
+metadata:
+  name: health-checks
+  description: Deploys a simple pod to test health checks
+
+components:
+  - name: health-checks
+    optional: false  # Changed from `required: true`
+    manifests:
+      - name: ready-pod
+        namespace: health-checks
+        wait: false  # Changed from `noWait: true`
+        files:
+          - ready-pod.yaml
+    actions:
+      onDeploy:
+        after:
+          - wait:
+              cluster:
+                kind: Pod
+                name: ready-pod
+                namespace: health-checks
+                apiVersion: v1
+                # condition is empty, so Kstatus will be used for readiness check
+```
+
+### Risks and Mitigations
+
+<!--
+What are the risks of this proposal, and how do we mitigate? Think broadly.
+For example, consider both security and how this will impact the larger
+Zarf ecosystem.
+
+How will security be reviewed, and by whom?
+
+How will UX be reviewed, and by whom?
+-->
+
+The field `.components.[x].dataInjections` will be removed without a direct replacement in the schema. There must be documentation to present to users so they know what alternatives they can use to achieve a similar result. 
+
+The alpha field `.components.[x].charts.[x].variables` has seen significant adoption and there will be no automatic conversion to its replacement Zarf values. There must be documentation on how users can utilize Zarf values as an alternative to chart variables. 
```

Comment:
Agreed, this is primarily in scope for the values work, imo. There's some mention of migrating vars over in the currently proposed docs, but a specific migration guide is probably warranted.